addons/refuel/functions/fnc_connectNozzleAction.sqf

```diff
@@ -128,6 +128,7 @@ private _attachPosModel = _target worldToModel (ASLtoAGL _bestPosASL);
         };
 
         [_unit, _target, _nozzle, _endPosTestOffset] call FUNC(refuel);
```

Comment:
Add the following line here:
`_unit setVariable [QGVAR(tempFuel), nil];`

Before there was a pause before connecting and refueling when this var would be reset.
Without clearing it, it will use the tempFuel from the last fueled vic (which would probably be 1), so it would instantly fill vics.

ref https://github.com/acemod/ACE3/blob/master/addons/refuel/functions/fnc_refuel.sqf#L91